.github/workflow/sigoff-check/action.yml

```diff
@@ -0,0 +1,19 @@
+# Copyright (c) 2020, NVIDIA CORPORATION.
```

Comment:
The year should be '2021'.

Reply:
@firestarman I just changed all 2020 to 2021. How do you think now? thx